crates/re_space_view_spatial/src/parts/images.rs

```diff
@@ -273,7 +273,13 @@ impl ImagesPart {
                 meaning,
                 color.into(),
             ) {
-                self.extend_bbox(&textured_rect);
+                // Only update the bounding box if the image_plane_distance is not auto.
+                // This is because in auto mode the bounds will change the image plane heuristic,
+                // and there's no good reason to grow the size of the bounds only to the shrink
+                // the image plane as a result.
```

Comment:
More to the point: it causes a cyclic relationship, where the auto-distance depend on the bounding box, and the bounding box is affected by the distance

Reply:
It's only lightly cyclic though -- since net result is to shrink the plane it doesn't continue the cycle.